src/test/java/com/rocketden/main/service/GameManagementServiceTests.java

```diff
@@ -122,12 +135,12 @@ public void addGetAndRemoveGame() {
         user.setUserId(USER_ID);
         room.addUser(user);
 
+        // Return a non-empty list so at least one problem exists
+        Mockito.doReturn(Collections.singletonList(null)).when(problemService).getProblemsFromDifficulty(eq(room.getDifficulty()), eq(1));
```

Comment:
Does this mean that we don't verify that the problems in the list are non-null? Should we include a check for this (and change this to include an actual `Problem`), or no?

Reply:
I think we should be ok on the checking for null (since the majority of our service classes don't check for null when it's guaranteed the called method will return a non-null object) - however, I've changed this to be `Collections.singletonList(new Problem())` to make it less confusing!